tests/test_files/test_class_with_fixtures.py

```diff
@@ -0,0 +1,14 @@
+class TestClass:
```

Comment:
Why? We make flake-plugin for pytest, not for unittest. Also, we have an [issue](https://github.com/best-doctor/flake8-fine-pytest/issues/1) to forbid this type of syntax

Reply:
It's an open-source util, and pytest allows creating tests as class methods. Therefore, it has to be supported.